packages/scss/src/utilities/_spacing.scss

```diff
@@ -10,105 +10,105 @@
 /* Margin bottom */
 
```

Comment:
I understand, but I used these with my previous team during 1 or 2 years, and it was a huge time saving ad it avoid creating CSS sheets for many pages.

By removing them, it will lead to add more classes or modifiers, which mean that your components will have style in multiple style sources, especially if you have differents margins to apply on the same component for UI requirements. 

I'm pretty sure that we will lose time without them, and as it's only helpers, it's absolutely not an obligation to use it.